torch/distributed/_composable/fsdp/_fsdp_param.py

```diff
@@ -199,37 +199,35 @@ def _init_sharded_param(self, param: nn.Parameter, device: torch.device):
                     "FSDP requires the DP and TP mesh to have the same parent mesh but got: \n"
                     f"DP's global mesh: {dp_global_mesh}\nTP's global mesh: {tp_global_mesh}"
                 )
-            self._global_mesh = dp_global_mesh
+
+            name_dims_error = (
+                "Please name your devicemesh dims, required for named slicing"
```

Comment:
nit: maybe something like this?
```suggestion
                "FSDP requires named DeviceMesh dims for ND parallelism"
```